README.md

```diff
@@ -86,6 +88,31 @@ Note above that the call to `target_link_libraries()` is using PUBLIC
 linking. This is required to make sure the include directories from `c4core`
 are transitively used.
 
+### Bazel
+
+Add c4core to your `WORKSPACE.bazel`:
+
+```python
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
+    name = "c4core",
+    init_submodules = True,
+    tag = "v0.1.10",
```

Comment:
please ensure this is version-consistent in the future, by adding the following to the proper section in [tbump.toml](tbump.toml):
```toml
[[file]]
src = "README.md"
search = "v{current_version}"
```